openpdf/src/main/java/com/lowagie/text/Element.java

```diff
@@ -312,7 +315,7 @@ public interface Element {
      * @return a type
      */
 
-    int type();
+    public int type();
 
```

Comment:
Java interface is public by default, NO need to declare it as public, which makes code heavy.